pkg/services/rpcsrv/subscription_test.go

```diff
@@ -376,7 +395,16 @@ func TestFilteredSubscriptions(t *testing.T) {
 					}
 					continue
 				}
-				this.check(t, resp)
+				if this.shouldCheck {
+					checkCalled = true
+					this.check(t, resp)
+				} else {
+					t.Fatal("unexpected notification match for case: " + name)
```

Comment:
Case name is already provided by `t.Run`, no need to duplicate this information.

Reply:
Instead, serialized notification can be passed as an argument. It's just easier to debug in this case.

Reply:
Is event ID enough?

Reply:
Let it be.